src/lib/models/useDeleteModelMutation.ts

```diff
@@ -0,0 +1,73 @@
+import { useDataEngine, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { useMutation } from 'react-query'
+import { Schema } from '../useLoadApp'
+
+type MutationFnArgs = {
+    id: string
+    displayName: string
+    messages?: string[]
```

Comment:
What's this for? Messages is computed from errors?

Reply:
I've changed this slightly so it's more clear. There can be error reports on the response error which let the user know why an item couldn't be deleted (see the 2nd video I shared above)